### PR TITLE
Add "Canonical" item to top nav

### PR DESCRIPTION
**Pull request by minkyngkm**
This is merge pull request to the `top nav` branch

## Done

- Remove global nav 
- Create`canonical` menu in the main nav 
- Build dropdown under `canonical` button

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- [List additional steps to QA the new features or prove the bug has been resolved]

## Issue / Card

Fixes #

## Screenshots

image

## Help

[QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html) - [Commit guidelines](https://canonical-web-and-design.github.io/practices/workflow/commit-messages.html)

### CODE REVIEW

**Comment by davegoddard42** [comment]
@minkyngkm 

There needs to be some space between the header "Resources" and the item "MicroStack" (looks fine for the "About" section)

image

Also, there's no way of accessing the "Canonical" Menu on mobile. On mobile, it should look like this:

image

But if we're focusing on the top nav only, I'm not sure if that's out of scope. Maybe a catch up next week to discuss?

**Comment by davegoddard42** [comment]
@petesfrench Thanks Pete. The spacing is perfect.

I have mixed feelings about the "canonical" menu item. It does fix the problem, but I'm not a huge fan of the way it looks - it feels like a CTA. The current menu is awful anyway, and this is only temporary, so it might not matter. I'm going to share it within the design team and get some feedback first if you don't mind? 

I'll share in Design Assembly tomorrow morning and will have an answer either way by lunchtime.

**Comment by davegoddard42** [comment]
@petesfrench - OK, thanks for the update!

**Comment by petesfrench** [comment]
@anthonydillon yes, but I thought it was better to have the spacing exact.

File: templates/templates/navigation-canonical-h.html [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+            <a href="https://microstack.run/">MicroStack&nbsp;&rsaquo;</a>
+          </h4>
+          <div class="u-hide--small">
+            <p class="p-p--small">Single-node OpenStack for developers and IoT.</p>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="col-3">
+      <h4 class="p-muted-heading--small">RESOURCES</h4>
```
Comment (sowasred2012) [inline templates/templates/navigation-canonical-h.html:170]:
```suggestion
      <h4 class="p-muted-heading--small">Resources</h4>
```

File: templates/templates/navigation-canonical-h.html [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+        <li class="p-list__item p-p--small"><a href="https://ubuntu.com/blog?category=case-studies#posts-list">Case studies</a></li>
+        <li class="p-list__item p-p--small"><a href="https://ubuntu.com/blog?category=white-papers#posts-list">White papers</a></li>
+        <li class="p-list__item p-p--small"><a href="https://docs.ubuntu.com/">Docs</a></li>
+        <li class="p-list__item p-p--small"><a href="https://ubuntu.com/cloud/training">Training</a></li>
+        <li class="p-list__item p-p--small"><a href="https://ubuntu.com/blog">Blog</a></li>
+        <li class="p-list__item p-p--small"><a href="https://developer.ubuntu.com/">Developer</a></li>
+        <li class="p-list__item p-p--small"><a href="https://ubuntu.com/download/cloud">Install</a></li>
+        <li class="p-list__item p-p--small"><a href="https://ubuntu.com/download">Download</a></li>
+      </ul>
+      <h4 class="p-muted-heading--small">ABOUT</h4>
```
Comment (sowasred2012) [inline templates/templates/navigation-canonical-h.html:184]:
```suggestion
      <h4 class="p-muted-heading--small">About</h4>
```

File: templates/templates/navigation-canonical-h.html [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="dropdown-window__content">
+  <div class="u-fixed-width"><hr></div>
+  <div class="row">
+    <div class="col-9" style="margin-bottom:24px;">
```
Comment (sowasred2012) [inline templates/templates/navigation-canonical-h.html:103]: @anthonydillon is right, we should refrain from using both inline styles and pixel values wherever possible.

```suggestion
    <div class="col-9 u-sv3">
```

**Review by sowasred2012 — APPROVED** [review]
LGTM - 3 small changes in the code before merging.

**Pull request merged**